digit-health-services/health-delivery-service/src/main/java/org/digit/health/delivery/web/models/Delivery.java

```diff
@@ -0,0 +1,51 @@
+package org.digit.health.delivery.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRawValue;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Delivery extends CampaignData{
```

Comment:
We don't need this Delivery to extend CampaignData

Reply:
Done.